### PR TITLE
fix(readme): Add a test for empty file list in git hook.

### PR DESCRIPTION
**Pull request by Klafyvel**
As it was, the proposed git hook will fail if no Julia file has been edited in the commit (for example if we only edited Project.toml).

### CODE REVIEW

**Comment by fredrikekre** [comment]
Thanks. I spent some time trying the other week to only run on the staged code too, not sure if there is a better way than
```
git stash -k
runic ...
# verify result (and stash changes automatically?)
git pop
```

Do you have any thoughts or ideas here?

File: README.md [hunk 1]
```diff
@@ -213,7 +213,10 @@ exec 1>&2
 
 # Run Runic on added and modified files
 mapfile -t files < <(git diff-index --name-only --diff-filter=AM master | grep '\.jl$')
-julia --project=@runic -m Runic --check --diff "${files[@]}"
+
+if [ ! -z "${files}" ]; then
```
Comment (fredrikekre) [inline README.md:217]:
```suggestion
if [ ${#files[@]} -gt 0 ]; then
```

Comment (fredrikekre) [reply README.md]: (https://www.shellcheck.net/ complained about `${files}` only expanding the first element (although that would be fine in this case) and about the use of `! -z` instead of `-n` FYI)

**Comment by Klafyvel** [comment]
> Do you have any thoughts or ideas here? In particular I don't know if you are allowed to run git commands in a git hook

I think I remember reading that git hooks are not supposed to apply changes to the files, but I don't remember where.

File: README.md (hunk 1 above)
Comment (Klafyvel) [reply README.md]: oh, TIL something new then. :)
